core/include/thread.h

```diff
@@ -413,7 +413,7 @@ int thread_wakeup(kernel_pid_t pid);
  */
 static inline kernel_pid_t thread_getpid(void)
 {
-    extern volatile kernel_pid_t sched_active_pid;
+    extern kernel_pid_t sched_active_pid;
```

Comment:
Why is this extern declaration needed anyway? We do include sched.h ...

Reply:
You're right, it should not be needed. I'd guess it was needed once upon a time...